plugins/modules/object_keys.py

```diff
@@ -164,9 +168,31 @@ def _handle_key_absent(self) -> None:
             self._key.delete()
             self.register_action("Deleted key {0}".format(label))
 
+    def _attempt_warnings(self, **kwargs: Any) -> None:
+        """
+        Raises warnings depending on the user-defined module arguments.
+        """
+
+        # Logic to warn if the `cluster` field has been specified
+        access: Optional[List] = kwargs.get("access", None)
+
+        # If cluster has been defined for any of the `access` objects,
+        # raise a deprecation warning
+        if access is not None and any(
+            v is not None for v in access if v.get("cluster", None)
+        ):
```

Comment:
@zliang-akamai I think this deprecation will be dependent on TPT-2603, do you mind if  I mark this PR as do not merge until that work is done?